compute/instances/troubleshooting/cant-connect-ssh.mdx

```diff
@@ -133,4 +133,8 @@ You must upload the content of the public part of the SSH key pair to the Scalew
   If you have any difficulties connecting to an Instance after uploading a new public SSH key to your Project, try the following:
     - If you cannot connect to your Instance at all via SSH, reboot your Instance from the console and try again.
     - If you can connect to your Instance using a previously uploaded SSH key but not the new one, go ahead and connect to your Instance with the old key. Once connected, run the `scw-fetch-ssh-keys --upgrade` command, which launches a script on your Instance to update your SSH keys. You can then check that the new key has been added to the `authorized_keys` file (`~/.ssh/authorized_keys`). Note that this command works only for Instances.
-</Message>
\ No newline at end of file
+</Message>
+
+## Timeout when trying to connect
+
+You may find the SSH connection attempt times out without connecting. This is expected behavior if the Instance is attached to a Private Network on which there is also a Public Gateway advertising the default route. See our [dedicated troubleshooting](/network/public-gateways/troubleshooting/cant-connect-to-instance-with-pn-gateway/) page for more help with this issue.
```

Comment:
```suggestion
You may find the SSH connection attempt times out without connecting. This may be expected behavior if the Instance is attached to a Private Network on which there is also a Public Gateway advertising the default route. See our [dedicated troubleshooting](/network/public-gateways/troubleshooting/cant-connect-to-instance-with-pn-gateway/) page for more help with this issue.
```